Finder/FinderFactory.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Bazinga\Bundle\JsTranslationBundle\Finder;
+
+use Symfony\Component\Finder\Finder;
+
+class FinderFactory
```

Comment:
having such a factory is only useful to allow you mocking the Finder in tests, which is already something your test should not do.